From: Dermot Bradley <dermot_bradley@yahoo.com>
Date: Sat,  4 Dec 2022 22:37 +0000
Subject: [PATCH] build gotestsum so tests run

step-cli 0.23.0 started using gotestsum for tests but didn't actually
build the gotestsum program.

---

diff -aur a/Makefile b/Makefile
--- a/Makefile
+++ b/Makefile
@@ -88,7 +88,8 @@
 #########################################
 
 test:
-	$Q $(CGO_OVERRIDE) $(GOFLAGS) gotestsum -- -coverprofile=coverage.out -short -covermode=atomic ./...
+	$Q go install gotest.tools/gotestsum@latest
+	$Q $(CGO_OVERRIDE) ~/go/bin/gotestsum -- -coverprofile=coverage.out -short -covermode=atomic ./...
 
 race:
 	$Q $(CGO_OVERRIDE) $(GOFLAGS) gotestsum -- -race ./...
